release/release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Release script
+# ===============
+# Sets the new version for the package.json file, pom.xml and as a git tag.
+# Dependencies:
+# - xmlstarlet (sudo apt-get install xmlstarlet)
+
+versionIdentifier=$1
+if [[ ! $versionIdentifier =~ ^(v[0-9]+(\.[0-9]+)*|major|minor|patch)$ ]]; then
+    echo "Version identifier is missing or invalid. Possible values are: [<v[major.minor.patch]> | major | minor | patch]"
+    exit 1
+fi
+
+if [ -n "$(git status --porcelain)" ]; then
+  # Uncommitted changes
+  echo "Working directory is not clean. Please commit the changes before creating a release."
+  exit 1
+fi
+
+rootDir="$(dirname "$0")"/..
+frontendDir="${rootDir}"/frontend
+pomDir="${rootDir}"
+
+currentVersion=$(grep -oP '(?<="version": ")[^"]*' "${frontendDir}"/package.json)
```

Comment:
Is this how this is done?
Can't we just use something like this:
node -pe "require('./package.json').version"

Reply:
Yes, this would be a better solution since this script already requires Node.